py_zipkin/zipkin.py

```diff
@@ -639,6 +640,12 @@ def __init__(self, *args, **kwargs):
         super(zipkin_server_span, self).__init__(*args, **kwargs)
 
 
+def choose_sampling(sample_rate):
```

Comment:
nit: maybe call this `should_sample`? choose_sampling gives me the idea that it'll return the sample rate.

Also, I'd add a leading `_` to this function since it doesn't seem something that's useful outside of this code.

Reply:
Done